resources/dicts/events/death/general.json

```diff
@@ -3155,7 +3096,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c sneaks out of camp, wondering what all is out there. {PRONOUN/m_c/subject/CAP} {VERB/m_c/come/comes} to a small stream and {VERB/m_c/attempt/attempts} to leap across. {PRONOUN/m_c/subject/CAP} misjudged the distance and {PRONOUN/m_c/poss} paws scrabble on the bank for purchase. m_c is losing {PRONOUN/m_c/poss} grip when {PRONOUN/m_c/subject} {VERB/m_c/think/thinks} {PRONOUN/m_c/subject} {VERB/m_c/catch/catches} a glimpse of r_c's pelt. {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} about to call for help when {PRONOUN/m_c/poss} grip fails.",
+        "event_text": "m_c sneaked out of camp and misjudged the jump across a stream. r_c watched as {PRONOUN/m_c/subject} disappeared under the water.",
```

Comment:
snuck

Also unclear if r_c caused the death or just witnessed it

Reply:
Same for the OG event, but we could also workshop this one more if we want it to be more clear?

Reply:
I think 'watched without concern' would be enough to tip it into murder territory

Reply:
oooh, I like that! Yeah we can go with that